runtime/tests/third_party/dummy_device.toml

```diff
@@ -1,95 +1,91 @@
-schema = 1
+schema = 2
 
 [device]
 name = "dummy.device.qubit"
 
-[operators]
+[operators.observables]
 # Observables supported by the device
-observables = [
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "Hermitian",
-        "Identity",
-        "Projector",
-        "SparseHamiltonian",
-        "Hamiltonian",
-        "Sum",
-        "SProd",
-        "Prod",
-        "Exp",
-]
 
-# The union of all gate types listed in this section must match what
-# the device considers "supported" through PennyLane's device API.
-[[operators.gates]]
-native = [
-        "QubitUnitary",
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "MultiRZ",
-        "Hadamard",
-        "S",
-        "T",
-        "CNOT",
-        "SWAP",
-        "CSWAP",
-        "Toffoli",
-        "CY",
-        "CZ",
-        "PhaseShift",
-        "ControlledPhaseShift",
-        "RX",
-        "RY",
-        "RZ",
-        "Rot",
-        "CRX",
-        "CRY",
-        "CRZ",
-        "CRot",
-        "Identity",
-        "IsingXX",
-        "IsingYY",
-        "IsingZZ",
-        "IsingXY",
-]
+PauliX
+PauliY
+PauliZ
+Hadamard
+Hermitian
+Identity
+Projector
+SparseHamiltonian
+Hamiltonian
+Sum
+SProd
+Prod
+Exp
 
+[operators.gates.decomp]
 # Operators that should be decomposed according to the algorithm used
 # by PennyLane's device API.
 # Optional, since gates not listed in this list will typically be decomposed by
 # default, but can be useful to express a deviation from this device's regular
 # strategy in PennyLane.
-decomp = [
-        "SX",
-        "ISWAP",
-        "PSWAP",
-        "SISWAP",
-        "SQISW",
-        "CPhase",
-        "BasisState",
-        "QubitStateVector",
-        "StatePrep",
-        "ControlledQubitUnitary",
-        "DiagonalQubitUnitary",
-        "SingleExcitation",
-        "SingleExcitationPlus",
-        "SingleExcitationMinus",
-        "DoubleExcitation",
-        "DoubleExcitationPlus",
-        "DoubleExcitationMinus",
-        "QubitCarry",
-        "QubitSum",
-        "OrbitalRotation",
-        "QFT",
-        "ECR",
-]
 
+SX
+ISWAP
+PSWAP
+SISWAP
+SQISW
+CPhase
+BasisState
+QubitStateVector
+StatePrep
+ControlledQubitUnitary
+DiagonalQubitUnitary
+SingleExcitation
+SingleExcitationPlus
+SingleExcitationMinus
+DoubleExcitation
+DoubleExcitationPlus
+DoubleExcitationMinus
+QubitCarry
+QubitSum
+OrbitalRotation
+QFT
+ECR
+
+[operators.gates.matrix]
 # Gates which should be translated to QubitUnitary
-matrix = [
-        "MultiControlledX",
-]
+
+MultiControlledX
+
+[operators.gates.native]
```

Comment:
Native gates first still makes the most sense to me

Reply:
@dime10 could you please elaborate? Did you mean the order of sections?

Reply:
Ah yes that's what I meant. Not a big deal but listing the important gates first (which are the natives ones) makes the most sense to me.

Reply:
Ok, done